src/Core/tests/DeviceTests/Handlers/Slider/SliderHandlerTests.Windows.cs

```diff
@@ -48,6 +112,26 @@ public async Task SmallChangeInitializesCorrectly(double min, double max, double
 			GetNativeSlider(sliderHandler).Maximum;
 
 		double GetSmallChange(SliderHandler sliderHandler) =>
-		GetNativeSlider(sliderHandler).SmallChange;
+			GetNativeSlider(sliderHandler).SmallChange;
+
+		Size GetNativeThumbSize(SliderHandler sliderHandler)
+		{
+			var nativeSlider = GetNativeSlider(sliderHandler);
+			
+			if(nativeSlider.GetFirstDescendant<Thumb>() is Thumb thumb)
+			{
+				return new Size(thumb.Width, thumb.Height);
+			}
+
+			return Size.Zero;
+		}
+
+		bool ImageSourceLoaded(SliderHandler sliderHandler)
+		{
+			if (sliderHandler.PlatformView is MauiSlider mauiSlider)
+				return mauiSlider.ThumbImageSource != null;
+
+			return false;
```

Comment:
We can simplify this:
```suggestion
           return (sliderHandler.PlatformView as MauiSlider)?.ThumbImageSource != null;
```